src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -17,11 +13,7 @@
 
 import seedu.address.logic.commands.AddCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.patient.Address;
-import seedu.address.model.patient.Email;
-import seedu.address.model.patient.Name;
-import seedu.address.model.patient.Patient;
-import seedu.address.model.patient.Phone;
+import seedu.address.model.patient.*;
```

Comment:
Also I believe that the coding standard guide says that we should list the imports explicitly instead of using wildcard imports? If not then it's ok

Reply:
Got it, java have the tendency to change all related imports to a single *, will update it!!